### ISSUE TITLE
As a developer, I want to interact with AWS as a project user so that we can maintain security

### ISSUE THREAD

**Issue by baskaufs** [HeardLibrary/cloud-tvnews#12]
I created the stateMachineUser but it has admin privileges, which probably isn't good.  Should it actually have login abilities?  Can it have weaker privs?  Just not sure what the best practice is here...

**Comment by baskaufs** [comment]
See best practices here:
https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html
It seems that the reason for creating another user is that it's just easier to delete the account if necessary.  That is difficult to do with the root user.

**Comment by baskaufs** [comment]
This is also intereting:
https://wblinks.com/notes/aws-tips-i-wish-id-known-before-i-started/

**Comment by CliffordAnderson** [comment]
@baskaufs Agreed! I think this is excellent advice: "Everyone gets an IAM account. Never login to the master."

**Comment by baskaufs** [comment]
OK, I think this is going to be my last comment with a link to a web post.  This is also relevant to issue #3 with respect to the issue of whether it's necessary to keep our account number a secret. 
https://adeliarisk.com/secure-cloud-computing-7-ways-id-hack-aws/
Notice that the AWS account number is not mentioned.  The security risks mentioned are generally related to insecure passwords and accidentally making secret keys available publicly.   I think we have already recognized these issues and have implemented 2FA, which is one of their suggestions.
